I.write-tests/09-test-async/async-1.test.js

```diff
@@ -1,3 +1,5 @@
+//A bit confused as to what this code is doing. It all passed the test without any change from me.
```

Comment:
Hmm yeah this is a bit vague here in terms of instructions. Do you follow the code though and understand what it's doing? What part of it are you confused about?